app/models/ability.rb

```diff
@@ -4,6 +4,7 @@ class Ability
   def initialize(user)
     if user
       can %i(create read destroy update), Wishlist, user_id: user.id
+      can :manage, Item, :wishlist => { :user_id => user.id }
```

Comment:
Why :manage? You are only doing  new and create right now. But it's not so important.